code/game/dna/dna_modifier.dm

```diff
@@ -125,6 +125,10 @@
 	add_fingerprint(usr)
 	return
 
+/obj/machinery/dna_scannernew/Destroy()
+	eject_occupant()
+	. = ..()
```

Comment:
😦

Reply:
oh yeah. Me being lazy and using the default stuff VS code generates. Will change